src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-thu">
+      <div class="calendar__day"> </div>
```

Comment:
no needs to add a space
```suggestion
      <div class="calendar__day"></div>
```